app/forms/firewall-rules-common.tsx

```diff
@@ -132,8 +133,10 @@ const DynamicTypeAndValueFields = ({
           items={items}
           allowArbitraryValues
           hideOptionalTag
-          // TODO: validate here, but it's complicated because it's conditional
-          // on which type is selected
+          validate={(value) =>
+            // TODO: is required false correct here? should this function even have that argument?
+            validateName(value, `${capitalize(sectionType)} name`, false)
+          }
```

Comment:
I think `required: false` is correct because we don't want this to show a required error when it's left empty after a bad input was attempted.